src/m365/base/ContextCommand.ts

```diff
@@ -0,0 +1,5 @@
+import AnonymousCommand from './AnonymousCommand';
+
+export default abstract class ContextCommand extends AnonymousCommand {
```

Comment:
If we're not adding any functionality here, we should remove this base class and inherit the command directly from AnonymousCommand.